src/sc/sidechainrpc.h

```diff
@@ -77,6 +80,45 @@ void fundCcRecipients(const CTransaction& tx, std::vector<CcRecipientVariant>& v
 void AddScInfoToJSON(UniValue& result);
 bool AddScInfoToJSON(const uint256& scId, UniValue& sc);
 void AddScInfoToJSON(const uint256& scId, const ScInfo& info, UniValue& sc);
+
+// Input UTXO is a tuple (triple) of txid, vout, amount)
+typedef std::tuple<uint256, int, CAmount> CmdUTXO;
```

Comment:
I have two objections to this typedef:
1- It is only relevant within scRpcCreationCmd now, so I would move it within scRpcCreationCmd  declaration, even with private access
2- It's not really a command, rather a data type. How about renaming it to something like selectedUTXO?

Reply:
Ok to both